docs/iqe-ee-proxy.md

```diff
@@ -26,6 +26,12 @@ https://insights-qe.pages.redhat.com/iqe-core-docs/tutorial/part3.html#download-
 `oc login …`
 `bonfire namespace reserve`
 `bonfire deploy rhsm …`
+
+Don't forget to deploy the additional "helper" containers (e.g. wiremock, kafka-bridge, mock prometheus), since they're EE specific and defined outside of the rhsm clowdapp.
+`oc process -f ../rhsm-subscriptions/stub/wiremock.yaml | oc apply -f -`
+`oc process -f ../rhsm-subscriptions/kafka-bridge/deploy/template.yaml | oc apply -f -`
+`oc process -f ../rhsm-subscriptions/swatch-metrics/deploy/mock-prometheus-clowdapp.yaml | oc apply -f -`
```

Comment:
I never encountered this issue because when I deploy rhsm, I use the ephemeral profile which already includes wiremock, kafka bridge and so on. 
But I acknowledge that this is really helpful when simply doing `bonfire deploy rhsm --no-remove-resources app:rhsm` (which uses the latest production image and templates)

Reply:
Oh good point, I didn't even think about using the ephemeral profile.  I was just using that simple command.

Reply:
Where is the documentation on deploying rhsm to ephemeral using the ephemeral profile? There should probably be a link from the beginning of this doc to wherever that is (or include it in this doc) so that the requirements for the setup are linked together.